krux_marathon_api/marathonapi.py

```diff
@@ -107,86 +96,70 @@ def assign_config_data(self, config_file_data, marathon_app_result):
         marathon_app_result.user                     = config_file_data["user"]                 #updates confirmed
 
     def list_marathon_apps(self, marathon_server):
+        """
+        This method connects to the marathon server and requests all the
+        Marathon Apps.
+        """
         self.logger.info("Listing apps running on marathon")
         current_marathon_apps = marathon_server.list_apps()
```

Comment:
Nit: You might want to list out the names of the apps found here in `self.logger.debug()`.